lib/missing_check_in_notifier.rb

```diff
@@ -9,7 +9,9 @@ def self.inactive_accounts
 
   def self.notify
     inactive_accounts.each do |account|
-      MissingCheckInMailer.missing_check_in(account, 1.day.ago).deliver_now
+      if account.features.check_in = true
```

Comment:
You're assigning the value here instead of checking. I believe you can do `account.features.check_in?`